pkg/kv/kvserver/replica_rangefeed.go

```diff
@@ -221,6 +221,13 @@ func (tp *rangefeedTxnPusher) Barrier(ctx context.Context) error {
 	return nil
 }
 
+var rangeFeedBulkDeliveryEnabled = settings.RegisterBoolSetting(
+	settings.SystemOnly,
+	"kv.rangefeed.bulk_delivery.enabled",
+	"if set, rangefeed may will use bulk delivery to send updates to clients during scans",
+	metamorphic.ConstantWithTestBool("rangefeed-bulk_delivery", false),
```

Comment:
Given where we are in the release cycle, I wouldn't be opposed to making this default on for a bit.

Reply:
Yeah, I intend to flip it in a standalone PR once we have a little more testing.